base/math.jl

```diff
@@ -18,14 +18,19 @@ export sin, cos, tan, sinh, cosh, tanh, asin, acos, atan,
        hankelh1, hankelh2, hankelh1x, hankelh2x,
        besseli, besselix, besselk, besselkx, besselh,
        beta, lbeta, eta, zeta, polygamma, invdigamma, digamma, trigamma,
-       erfinv, erfcinv, @evalpoly
+       erfinv, erfcinv, @evalpoly,
+       acos_fast, acosh_fast, asin_fast, asinh_fast, atan2_fast,
```

Comment:
Maybe these functions would better not be exported to avoid cluttering up the workspace, and only be available via `@fastmath`?